src/app.ts

```diff
@@ -30,7 +30,14 @@ MongoDriverFactory.build()
         name 
       });
       res.json ({id});
-    })
+    });
+
+      //delete a specific task 
+      app.delete('/tasks/:id', async (req, res) => {
```

Comment:
Be sure to mind indentation levels. It looks like this is nested in a little too far